### ISSUE TITLE
Texture loads only when player is near the planets

### ISSUE THREAD

**Issue by Apobu** [Gameslinx/Tessellation#70]
Is there a way to have the parallax textures apearing the whole time? For example when in map view, stock textures apear. However when a craft goes near a planet parallax textures apear again (If landed on a planet and un-zoom, if that's a word, far enough it goes back to stock). I am almost 100% sure it's a setting thing but how can i change that? It didn't use to be like this. I just updated everything after 1.12.2 came out and this is how ot works now. I am guessing it's for perfomance issues but my pc can handle it i guess so.. why not. Anyway if you can help that would be great and thank you so much, this is the best mod for the game period.. <3

**Comment by Gameslinx** [comment]
Hi, planets use two view levels. One is called the PQS view - the physical view of the terrain. The other is called the Scaled view, which is a very rough model of the terrain. The scaled view is visible from space and the PQS view fades in when you get close to the planet.

I don't want the parallax textures to be visible from space - I'm working on an add-on that improves the Scaled textures that'll become part of Parallax when it's done

**Comment by Apobu** [comment]
Hi, I just want to say, you are awesome and I’m really exited for everything you might be working on. 
I just believe parallax textures are so good, there could be at least a config setting to choose whether the game loads using parallax textures both in PQS view and in Scaled view. Whatever the case might be though, your work is amazing and has made everything so much better.

P.s. After I posted the question you just replied to, I noticed my kopernicus mod wasn’t working because there is no support for 1.12 yet and I wasn’t seeing any parallax textures whatsoever. Probably that’s why everything looked so… old, haha. I’ll just wait for next kopernicus bleeding edge update. 

Thank you so much, I hope you’re we having a nice day 

>
